src/app/static/src/tests/components/projects/versionStatus.test.ts

```diff
@@ -20,16 +20,17 @@ describe("Version status component", () => {
             }
         });
         registerTranslations(store);
-        return shallowMount(VersionStatus, {store});
+        return shallowMountWithTranslate(VersionStatus, store, {global: {plugins: [store]}});
     };
     const date = new Date(2020, 8, 1, 12, 45, 59);
 
     it("does not render if no current project", () => {
         const wrapper = getWrapper();
-        expect(wrapper.isEmpty()).toBe(true);
+        // just empty element if v-if isn't true
+        expect(wrapper.html()).toBe("<!--v-if-->");
```

Comment:
Feels like there should be a nicer way of doing this, but not sure what it would be! I wonder why they got rid of `isEmpty`..

Reply:
yh I thought so too, looked around a bit and just found this really, not much online about it! yh kinda annoying that they decided to get rid of that